src/lib/chains/solana/solanaError.ts

```diff
@@ -0,0 +1,25 @@
+export enum SolanaError {
+    CLIENT_NOT_SET = 'CLIENT HAS NOT SET',
```

Comment:
Same issue here, please do not use all capitals in error string.

Reply:
Also, please check grammer:  

Client has not been set (or: Client not set)
Bridge has not been set (or Bridge not set)